module/observations/helper.js

```diff
@@ -1048,4 +1052,175 @@ module.exports = class ObservationsHelper {
         });
     }
 
+
+         /**
+      * observation link.
+      * @method
+      * @name getObservationLink
+      * @param  {String} observationSolutionId observation solution external Id.
+      * @param  {String} appName name of app.
+      * @returns {getObservationLink} observation getObservationLink.
+     */
+
+    static getObservationLink(observationSolutionId, appName) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                let link = null;
+                
+                let observationData = await solutionHelper.solutionDocuments({
+                        externalId : observationSolutionId,
+                        isReusable : false,
+                        type : messageConstants.common.OBSERVATION
+
+                        },[
+                            "link"
+                    ]);
+
+                if(!observationData || observationData == "") {
+                    return resolve({
+                        message: messageConstants.apiResponses.OBSERVATION_NOT_FOUND,
+                        result: {}
+                    });
+                }
+
+                let appDetails = await kendraService.getAppDetails(appName);
+                
+                if(appDetails.result === false){
+                    throw new Error(messageConstants.apiResponses.APP_NOT_FOUND);
+                }
+                
+                link = process.env.OBSERVATION_SHARE_URL_ENDPOINT+observationData[0].link;
```

Comment:
@priyanka-TL - Use APP_PORTAL_BASE_URL env variable here, similar to polls helper